clients/admin-ui/src/features/system/AddMultipleSystems.tsx

```diff
@@ -0,0 +1,252 @@
+/* eslint-disable react/no-unstable-nested-components */
+import {
+  Badge,
+  Button,
+  Flex,
+  Spinner,
+  Tooltip,
+  useDisclosure,
+  useToast,
+} from "@fidesui/react";
+import {
+  createColumnHelper,
+  getCoreRowModel,
+  getFilteredRowModel,
+  getPaginationRowModel,
+  getSortedRowModel,
+  useReactTable,
+} from "@tanstack/react-table";
+import { useRouter } from "next/router";
+import { useMemo, useState } from "react";
+
+import { useAppSelector } from "~/app/hooks";
+import ConfirmationModal from "~/features/common/ConfirmationModal";
+import { useFeatures } from "~/features/common/features";
+import {
+  DefaultCell,
+  DefaultHeaderCell,
+  FidesTableV2,
+  GlobalFilterV2,
+  IndeterminateCheckboxCell,
+  PaginationBar,
+  RowSelectionBar,
+  TableActionBar,
+  TableSkeletonLoader,
+} from "~/features/common/tablev2";
+import { successToastParams } from "~/features/common/toast";
+import {
+  DictSystems,
+  selectAllDictSystems,
+  useGetAllCreatedSystemsQuery,
+  usePostCreatedSystemsMutation,
+} from "~/features/plus/plus.slice";
+
+export const VendorSourceCell = ({ value }: { value: string }) => (
+  <Flex alignItems="center" height="100%">
+    <Badge>{value.split(".")[0] === "gvl" ? "GVL" : "AC"}</Badge>
```

Comment:
ah we're going to be doing this kind of operation pretty often I'm imagining—we'll probably want it as its own function eventually (see https://github.com/ethyca/fides/blob/main/clients/fides-js/src/lib/tcf/vendors.ts/#L10-L25 for reference)